x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/privilege/ClusterPrivilegeResolver.java

```diff
@@ -148,6 +148,9 @@
     public static final NamedClusterPrivilege MANAGE_LOGSTASH_PIPELINES = new ActionClusterPrivilege("manage_logstash_pipelines",
         Set.of("cluster:admin/logstash/pipeline/*"));
 
+    public static final NamedClusterPrivilege CANCEL_TASKS = new ActionClusterPrivilege("cancel_tasks",
```

Comment:
Recently we're trying to name these in the singular declension, ie cancel_tasks -> cancel_task .

Reply:
renamed, thanks